packages/amqp/lib/AmqpExchangePublisherManager.ts

```diff
@@ -0,0 +1,96 @@
+import { AbstractPublisherManager } from '@message-queue-toolkit/core'
+import type { MessageMetadataType } from '@message-queue-toolkit/core/lib/messages/baseMessageSchemas'
+import type { TypeOf } from 'zod'
+import type z from 'zod'
+import { util } from 'zod'
+
+import type {
+  AbstractAmqpExchangePublisher,
+  AMQPExchangePublisherOptions,
+} from './AbstractAmqpExchangePublisher'
+import type { AmqpQueueMessageOptions } from './AbstractAmqpQueuePublisher'
+import type { AMQPCreationConfig, AMQPDependencies, AMQPLocator } from './AbstractAmqpService'
+import type {
+  AmqpAwareEventDefinition,
+  AmqpMessageSchemaType,
+  AmqpPublisherManagerDependencies,
+  AmqpPublisherManagerOptions,
+} from './AmqpQueuePublisherManager'
+import { CommonAmqpExchangePublisherFactory } from './CommonAmqpPublisherFactory'
+
+import Omit = util.Omit
```

Comment:
🟢 nit: Importing omit could cause confusing with TS Omit, I would prefer using `util.Omit` or even `z.util.Omit` to make it more specific. But this is a very nitpicky comment, feel free to ignore

Reply:
good catch, this was autoresolve by WebStorm, not intentional. I wanted to use TS Omit